Formula/openssh.rb

```diff
@@ -34,6 +34,9 @@ class Openssh < Formula
   def install
     ENV.append "CPPFLAGS", "-D__APPLE_SANDBOX_NAMED_EXTERNAL__"
 
+    # Ensure sandbox profile prefix is correct.
+    inreplace "sandbox-darwin.c", "@PREFIX@/share/openssh", etc/"ssh"
```

Comment:
It seems bizarre that MacPorts and Homebrew are both having to patch a file named sandbox-darwin.c but I may be misunderstanding the situation.